Source/WebKitLegacy/mac/WebView/WebPreferencesDefaultValues.mm

```diff
@@ -207,6 +207,12 @@ bool defaultShouldConvertInvalidURLsToBlank()
     return shouldConvertInvalidURLsToBlank;
 }
 
+bool defaultInvokeAttributeEnabled()
```

Comment:
You probably don't need this and `SDKAlignedBehavior::InvokeAttributeEnabled`. Popover needs this because the `popover` attribute caused compat issues (but not the target attributes), so this method disables it on older apps. I suspect invoke target attributes don't need that compat treatment though.